src/ExadelMentorship.BusinessLogic/Features/WeatherFeature/FutureWeather/FutureWeatherService.cs

```diff
@@ -0,0 +1,90 @@
+﻿using ExadelMentorship.BusinessLogic.Exceptions;
+using ExadelMentorship.BusinessLogic.Interfaces.Weather;
+using ExadelMentorship.BusinessLogic.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ExadelMentorship.BusinessLogic.Features.WeatherFeature.FutureWeather
+{
+    public class FutureWeatherService : IFutureWeatherService
```

Comment:
You can combine this class and its interface with `CurrentWeatherService`. You can have single facade to work with Open Weather API.